src/node_dotenv.cc

```diff
@@ -134,13 +134,25 @@ void Dotenv::ParseContent(const std::string_view input) {
       auto newline = content.find('\n');
       if (newline != std::string_view::npos) {
         content.remove_prefix(newline + 1);
+        content = trim_spaces(content);
         continue;
+      } else {
+        break;
       }
     }
 
-    // If there is no equal character, then ignore everything
+    // Find the next equals sign and newline
     auto equal = content.find('=');
-    if (equal == std::string_view::npos) {
+    auto newline = content.find('\n');
```

Comment:
So, right now you're iterating the string twice, even though you can cut corners and iterate on equal first, and if it doesn't exist, you can break the loop. Iterating as less as possible would be a good win, and a good exercise IMHO.

I also recommend using content.find_first_of("=\n") but the performance is somewhat similar to what you're doing right now.

Also, "if there is no equal character in this line" check is a lot simpler now. If equal_or_newline is npos, and it is equal to newline character, this means there is no equal case...